src/client/src/app/components/login/login.component.ts

```diff
@@ -9,12 +15,22 @@ import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 
 export class LoginComponent implements OnInit {
   loginForm: FormGroup;
-    constructor( private fb: FormBuilder) { 
+  loginFailMsg$: Observable<string>;
+    constructor( private fb: FormBuilder, private authService: AuthService, private store: Store<AppState>) {
```

Comment:
Since the authService is not used in this component, we do not need it here, and also can remove it from the import at line 5